The job **`pr-check-2`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/26093713682/job/76726042547) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/76726042547) [(plain)](https://github.com/rust-lang/rust/commit/dcfde7c8c6567a799706c7d0a8142fd5851ed833/checks/76726042547/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
---- library/core/src/fmt/num_buffer.rs - fmt::num_buffer::NumBuffer (line 40) stdout ----
error[E0425]: cannot find value `n` in this scope
  --> library/core/src/fmt/num_buffer.rs:48:12
   |
48 | assert_eq!(n.format_into(&mut buf), "1972");
   |            ^
   |
help: a local variable with a similar name exists
   |
48 | assert_eq!(n1.format_into(&mut buf), "1972");
   |             +

error[E0425]: cannot find value `n` in this scope
  --> library/core/src/fmt/num_buffer.rs:52:12
   |
52 | assert_eq!(n.format_into(&mut buf), "-1972");
   |            ^
   |
help: a local variable with a similar name exists
   |
52 | assert_eq!(n1.format_into(&mut buf), "-1972");
   |             +

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0425`.
```

</details>

--- a/library/core/src/fmt/num_buffer.rs
+++ b/library/core/src/fmt/num_buffer.rs
@@ -34,6 +34,22 @@ impl_NumBufferTrait! {
 
 /// A buffer wrapper of which the internal size is based on the maximum
 /// number of digits the associated integer can have.
+///
+/// # Examples
+///
+/// ```
+/// #![feature(int_format_into)]
+/// use core::fmt::NumBuffer;
+///
+/// let mut buf = NumBuffer::new();
+/// let n1 = 1972u32;
+/// assert_eq!(n1.format_into(&mut buf), "1972");
+///
+/// // Formatting a negative integer includes the sign.
+/// let mut buf = NumBuffer::new();
+/// let n2 = -1972i32;
+/// assert_eq!(n2.format_into(&mut buf), "-1972");
+/// ```
 #[unstable(feature = "int_format_into", issue = "138215")]
 pub struct NumBuffer<T: NumBufferTrait> {
     // FIXME: Once const generics feature is working, use `T::BUF_SIZE` instead of 40.

